class/Defaults/Trigger/Plugin/Activated.php

```diff
@@ -0,0 +1,62 @@
+<?php
+/**
+ * WordPress Plugin activated trigger
+ *
+ * @package notification
+ */
+
+namespace BracketSpace\Notification\Defaults\Trigger\Plugin;
+
+use BracketSpace\Notification\Defaults\MergeTag;
+use BracketSpace\Notification\Abstracts;
+
+/**
+ * Active Plugin trigger class
+ */
+class Activated extends PluginTrigger {
+
+	/**
+	 * Constructor
+	 */
+	public function __construct() {
+
+		parent::__construct( array(
+			'slug' => 'wordpress/active_plugin',
+			'name' => __( 'Plugin Active', 'notification' )
+		) );
+
+		$this->add_action( 'activated_plugin', 10, 2 );
+		$this->set_group( __( 'WordPress', 'notification' ) );
+		$this->set_description( __( 'Fires when plugin is activated', 'notification' ) );
+
+	}
+
+	/**
+	 * Gets specific update type
+	 *
+	 * @param  string $plugin_rel_path Plugin path.
+	 * @return void
+	 */
+	public function action( $plugin_rel_path ) {
+		$plugin_dir = WP_PLUGIN_DIR.DIRECTORY_SEPARATOR.$plugin_rel_path;
```

Comment:
All `=` should be aligned

Reply:
Concatenation missing spacing, ie. `WP_PLUGIN_DIR . DIRECTORY_SEPARATOR . $plugin_rel_path;`